### PR TITLE
feat(arch): add tenant API

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

Fixes #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zjj2wry** [comment]
/hold

should be merge  this after i fix tenant-admin dependency

File: pkg/apis/cnetworking/v1alpha1/type.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	metav1.ListMeta `json:"metadata,omitempty"`
+
+	Items []NetworkPolicy `json:"items"`
+}
+
+// +genclient
+
+// This is a resource which is subset of NetworkPolicy of networking.caicloud.io
+// see github.com/kubernetes/kubernetes/pkg/apis/networking/v1/types.go
+type NetworkPolicy struct {
```
Comment (zoumo) [inline pkg/apis/cnetworking/v1alpha1/type.go]: what is this used for?

**Comment by zjj2wry** [comment]
/hold cancel
@zoumo @kdada 
拆成了两个 commits，PTAL

File: pkg/apis/cnetworking/v1alpha1/type.go (hunk 1 above)
Comment (zjj2wry) [reply pkg/apis/cnetworking/v1alpha1/type.go]: used tenant network isolation

**Pull request merged**